src/components/Map.svelte

```diff
@@ -32,7 +32,7 @@
 
   $: mapId = `${map.id}-${map.index}`;
   $: mapPrecompileOption = map?.selectedPrecompileOption;
-  $: mapLandmarks3D = map?.mapLandmarks3D;
+  $: mapLandmarks3D = map?.landmarks3D;
```

Comment:
@ebrelsford good catch! I had a typo where I was looking for a nonexistent key named after the variable I was trying to set 😓

Reply:
I missed this because I was testing on something that precompiled which was running every time regardless. Without the precompile this was causing it to not trigger.